blazor/common/best-practices.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: post
+title: Best Practices for Improving Syncfusion Blazor App Performance | Syncfusion
+description: Checkout and learn here all about optimal strategies for enhancing Syncfusion Blazor App performance.
+platform: Blazor
+component: Common
+documentation: ug
+---
+
+# Best Practices for Improving Syncfusion Blazor App Performance
+
+Enhancing the rendering performance of Syncfusion Blazor apps involves implementing the following best practices.
+
+## Use Individual NuGet Packages
+
+Opt for individual Syncfusion NuGet packages tailored to specific components. This approach allows you to selectively include only the necessary components, minimizing overhead. For detailed information, refer to [Benefits of Using Individual NuGet Packages](https://blazor.syncfusion.com/documentation/nuget-packages#benefits-of-using-individual-nuget-packages).
+
+## Reduce Script Reference Size
+
+The syncfusion-blazor-base.min.js' script reference may be redundant. Instead, consider employing the following solutions to include only the required scripts for rendering Syncfusion components.
```

Comment:
Is this line correct?

Reply:
The page has been corrected by replacing ‘syncfusion-blazor-base.min.js’ with overall ‘syncfusion-blazor.min.js’.